app/gui/src/main/java/org/mobilitydata/gtfsvalidator/app/gui/Main.java

```diff
@@ -0,0 +1,79 @@
+package org.mobilitydata.gtfsvalidator.app.gui;
+
+import com.google.common.flogger.FluentLogger;
+import java.awt.Desktop;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
+
+/**
+ * The main entry point for the GUI application.
+ *
+ * <p>Compared to the CLI jar, this entry point is designed to be packaged as a native application
+ * to be run directly by the user.
+ *
+ * <p>TODO(bdferris): Follow up work will add a minimal UI for selecting the input GTFS and
```

Comment:
If we're going to merge this PR as a first cut, should this TODO move to a GitHub issue?

Reply:
Updated the TODO in 2251f9435a14c1b890d495f708ef23dd118b969c to reference #1134.